### ISSUE TITLE
PromptChar doesn't change prompt char in some cases

### ISSUE THREAD

**Issue by musm** [nickcox/pure-pwsh#9]
e.g.
`
 $pure.PromptChar = "$"

then type e.g.
```
$ +   (changes to default char ??)

```

**Comment by nickcox** [comment]
I can't repro this. What version of PSReadLine are you using? I know that v1.x can be fussy about certain characters when it redraws the prompt (i.e. if you're working inside a git repository). I think `$` should be fine though.

Any chance of a screen shot?

**Comment by NeedsAdjustment** [comment]
https://cdn.discordapp.com/attachments/437996838541459486/563916462960410625/unknown.png

I'm using Alacritty with Powershell 6

**Comment by nickcox** [comment]
Thanks, I see what you mean. I think this is an issue with alacritty and PSReadLine. If I unload `pure-pwsh` I still have the same issue.

```
~
❯ Remove-Module pure-pwsh
PS> +
??
...
```

(Note that `Get-PSReadLineOption` still uses the `PromptText` and `ContinuationPrompt` as set by `pure-pwsh`.)

I think there are a couple of workarounds here:

1) Assuming you're already using PSReadline 2.0 (which is bundled with Windows 10 1809 and above), try upgrading PSReadLine to 2.0-beta-3. I couldn't repro this issue there.
2) If you can live with an ugly continuation prompt, try `Set-PSReadLineOption -PromptText ''` ; `Set-PSReadLineOption -ContinuationPrompt '>>'`.

**Comment by musm** [comment]
Thanks for the clarification. Another problem I found is that the line continuation prompt does not  actually double up say `PromptChar` but  doubles up the default  to  `❯❯`, which is undesirable, when using a font such as the default consolas on windows which does not have the glyph for ❯

**Comment by tiqk-nick** [comment]
Yep, this needs to be fixed. In the meantime you could manually

```
Set-PSReadlineoption -ContinuationPrompt ...
```

**Comment by nickcox** [comment]
Yep, this needs to be fixed. In the meantime you could manually `Set-PSRealineOption -ContinuationPrompt [...]`.

**Comment by nickcox** [comment]
Continuation prompt issue should be fixed in [0.6.1](https://www.powershellgallery.com/packages/pure-pwsh/0.6.1).
